include/libtorrent/entry.hpp

```diff
@@ -95,8 +95,30 @@ namespace libtorrent
 			bool operator()(T1 const& rhs, T2 const& lhs) const
 			{ return rhs < lhs; }
 		};
-	};
+
+		template<class T>
+		class map_string : public std::map<std::string, T, aux::strview_less>
```

Comment:
this could be a ``using`` statement.
``template <typename T>
map_string = std::map<std::string, T, aux::strview_less>;``